boot/boot.go

```diff
@@ -395,3 +395,41 @@ func SetRecoveryBootSystemAndMode(dev Device, systemLabel, mode string) error {
 	}
 	return bl.SetBootVars(m)
 }
+
+// UpdateManagedBootConfigs updates managed boot config assets if those are
+// present for the ubuntu-boot bootloader. Returns true when an update was
+// carried out.
+func UpdateManagedBootConfigs(dev Device) (updated bool, err error) {
+	if !dev.HasModeenv() {
+		// only UC20 devices use managed boot config
+		return false, ErrUnsupportedSystemMode
```

Comment:
this error is a bit strange for this function given that this function doesn't take a system mode as input, as the other user of this error

Reply:
perhaps this shouldn't error at all on non-uc20, same as there is no error when not using trusted assets bootloader